src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -78,11 +78,27 @@
      */
     private Map<String, DetectorData> detectors;
 
+    public Map<String, DetectorData> getDetectors(){
+        return this.detectors;
+    }
+
     /**
      * The time since the last update call.
      */
     private float timeSinceLastUpdate;
 
+    /**
+     * This provides the value for the test
+     * @return a float of timeSinceLastUpdate
+     */
+    public float getTimeSinceLastUpdate(){
```

Comment:
Again, parentheses. I am going to request that you go through every single method you added, and fix this.